doc/changelog.md

```diff
@@ -139,6 +139,8 @@
 * Add optimization that removes redundant chains of self inverse operations. This is done within a new MLIR pass called `remove-chained-self-inverse`. Currently we only match redundant Hadamard operations but the list of supported operations can be expanded.
   [(#630)](https://github.com/PennyLaneAI/catalyst/pull/630)
 
+* Update CI/CD workflow by using `manylinux_2_28_x86_64` instead of `manylinux_2014_x86_64` image [(#663)](https://github.com/PennyLaneAI/catalyst/pull/663)
```

Comment:
```suggestion
* Binary distributions for Linux are now based on `manylinux_2_28` instead of `manylinux_2014`. As a result, Catalyst will only be compatible on systems with `glibc` versions `2.28` and above (e.g. Ubuntu 20.04 and above).
  [(#663)](https://github.com/PennyLaneAI/catalyst/pull/663)
```

Reply:
Makes sense, yes. Done!